README.md

```diff
@@ -1 +1,85 @@
-# Covid-19-Vaccination-Tracker
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Meta TechDrip
+
+> A web app that fetches data from FakeStore API to display the list of products. The application allow users to check a list of different products and by clicking on a specific product, users can check the details of that product. Built with React, Redux Toolkit, API, CSS and JavaScript.
+
+
+## Built With
+
+- [x] React
+- [x] Redux
+- [x] [FakeStore API](https://fakestoreapi.com/)
+- [x] Stylelint and ESlint
+- [x] VS Code
+
+## Website Mockup 📱 💻 🖥️
+
+![capstone 3](https://user-images.githubusercontent.com/67911212/176967762-76f52e49-fb18-4b0f-bbc6-78d95fce884c.jpg)
+
+## Live Demo 🔗
+
+[Live Demo](https://62bf59a22da6092d8116c262--nimble-donut-8a2d42.netlify.app/)
+
+## Project Presentation
+
+[Video](https://www.loom.com/share/e2b7b6e9c7d2418b94bd091dfc23b4d9)
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+- [x] A web browser like Google Chrome
+- [x] A code editor like Visual Studio Code with Git and Node.js.
+
+### Setup
+
+Clone the repository using the GitHub [link](https://github.com/HammadSiddique/Meta-TechDrip.git)
+
+### Install
+
+In the terminal, go to your file directory and run this command.
+
+```
+$ git clone https://github.com/HammadSiddique/Meta-TechDrip.git
+```
+
+### Usage
+
+Kindly modify the files as needed.
+
+### Run tests
+
+To check for linters, use the files provided by [Microverse](https://github.com/microverseinc/linters-config). A GitHub action is also set to run during pull request.
+```
+$ npm install
+```
+
+## Authors
+
+👤 **Hammad Siddique** <img src="https://emojis.slackmojis.com/emojis/images/1531849430/4246/blob-sunglasses.gif?1531849430" width="20"/>
+
+  - [GitHub](https://github.com/HammadSiddique)
+  - [Twitter](https://twitter.com/hs_devv)
+  - [LinkedIn](https://www.linkedin.com/in/-hammadsiddique/)
+  
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/HammadSiddique/Meta-TechDrip/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Design idea [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio).
+- Microverse
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
[OPTIONAL] Kindly add an MIT license to your project. When I tried to open your license from Readme.MD it shows me a broken link.